### PR TITLE
Pub.Dev Packages Template

### PR DESCRIPTION
**Pull request by ksanty**
New checklist template for publishing packages to Pub.Dev

**- What I did**
Created a new publishing packages checklist template for Pub.Dev
**- How I did it**
Added to the at_widgets repo under .github directory
**- How to verify it**
Review the checklist and ensure its created properly and add any comments if you think we missed something
**- Description for the changelog**
 Initial version template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pubdev_checklist.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Referece Documents:
```
Comment (cpswan) [inline .github/ISSUE_TEMPLATE/pubdev_checklist.md]: Typo: s/Referece/Reference/

File: .github/ISSUE_TEMPLATE/pubdev_checklist.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+ [Pub.Dev List](https://docs.google.com/document/d/1GpHSJzQHTqzOpQlObOODWXloHoQRjy2-B1TRT-PMVlg/edit#)
```
Comment (cpswan) [inline .github/ISSUE_TEMPLATE/pubdev_checklist.md]: Better to just have the contents of the document here than send people on a treasure hunt.

Comment (ksanty) [reply .github/ISSUE_TEMPLATE/pubdev_checklist.md]: Will do.  List added below.

**Review by cpswan — APPROVED** [review]

**Comment by cpswan** [comment]
@sarika01 this seems to reflect the discussion from yesterday, but not merging yet so that you have a chance to review.

**Review by sarika01 — APPROVED** [review]

**Pull request merged**